test/modules/post/test/file.rb

```diff
@@ -32,8 +32,8 @@ def initialize(info={})
   #
   def setup
     @old_pwd = pwd
-    tmp = (directory?("/tmp")) ? "/tmp" : "%TMP%"
-    vprint_status("Setup: changing working directory to #{tmp}")
+    tmp = (directory?("/tmp")) ? "/tmp" : "%TEMP%"
+    puts "Setup: changing working directory to #{tmp}"
```

Comment:
What's this about?  `puts` is almost always wrong because it goes directly to stdout, whereas the `print_*` family go to whatever sink has been set up with `init_ui`, which might be a spool.